CHANGES.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 1.8.4 (core) / 0.24.4 (libraries)
+
+### Bugfixes
+
+- Fixed an issue where viewing run logs with the latest 5.0 release of the watchdog package raised an exception.
+- Fixed a bug that caused an "Asset must be part of at least one job" error when clicking on an external asset in the asset graph UI
```

Comment:
I think we also want to throw in this one:
- Fixes a bug that caused an error when loading the launchpad for a partition, when using in Dagster+ with an agent with version below 1.8.2.